streaming_ws_test.go

```diff
@@ -234,7 +234,7 @@ func TestHandleWS(t *testing.T) {
 }
 
 func TestDialRedirect(t *testing.T) {
-	client := NewClient(&Config{})
+	client := NewClient(&Config{}).NewWSClient()
 	_, err := client.dialRedirect(":")
 	if err == nil {
```

Comment:
I understand.
However, this line seems not to be wrong.
It wrong is the [264-265](https://github.com/mattn/go-mastodon/pull/39/files/010ec2eaf92da466640e155dfac5a9611b6a0f33#diff-7f77780830ccced377be5bedb5727d5aR264) line?